### DISCUSSION TITLE
Link extension

### DISCUSSION THREAD

**Discussion by crairdin** [ueberdosis/tiptap#2578] [General Questions] [answered]
I'm using tiptap in an Electron app, so I need control over click-handling. I also have custom links that I will satisfy within my app — they may or may not be any recognizable URL scheme. I just want to know the user has clicked on a link and what the href is and I'll take it from there. I can't seem to keep tiptap from following links, even if I set openOnClick to false. Because of the way Electron works, it essentially replaces my app's UI with the website being linked to. That won't go over well. :-)

**Comment by cadars** [comment]
Weird that `openOnClick` does not work… maybe add `target="_blank"` in `HTMLAttributes`?
Or `pointer-events:none` in CSS.

**Comment by crairdin** [comment, marked as answer]
My solution was to remove tiptap from my project and write my own replacement. Mine is either more full-featured or less, depending on your definition. My definition is that I don't have to write custom code for every HTML tag and attribute. :-)

**Comment by philippkuehn** [comment]
You have to write a plugin for that. Take a look at the default clickhandler plugin for the link extension as an inspiration: https://github.com/ueberdosis/tiptap/blob/main/packages/extension-link/src/helpers/clickHandler.ts
